src/WebJobs.Extensions.Http/Config/HttpExtensionConfigProvider.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks.Dataflow;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.WebJobs.Host.Config;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http
+{
+    /// <summary>
+    /// Defines the configuration options for the Http binding.
+    /// </summary>
+    public class HttpExtensionConfigProvider : IExtensionConfigProvider
```

Comment:
Does this have to be public anymore? I see other IExtensionConfigProvider implementations are non-public, e.g. TimersExtensionConfig, then others public like CosmosDBConfiguration. Hoping we can take this opportunity to get naming and visibility consistent so we have a recommended pattern.

Reply:
No, it doesn't. Changing to internal